app/templates/components/forms/user-profile-form.hbs

```diff
@@ -0,0 +1,43 @@
+<form class="ui form" autocomplete="off" {{action 'submit' on='submit'}}>
+  <div class="ui stackable grid">
+    <div class="four wide column">
+      {{widgets/forms/image-upload
+        imageUrl=model.avatarUrl
+        needsCropper=true
+        label=(t 'User Image')
+        id='user_image'
+        icon='photo'
+        hint=(t 'Select User Image')
+        maxSizeInKb=10000
+        helpText=(t 'We recommend using at least a 2160x1080px (2:1 ratio) image')}}
+      <div class="field">
+        <h3>Name {{model.firstName}}</h3>
+      </div>
+      <div class="field">
+        <h3>Family Name {{model.lastName}}</h3>
+      </div>
+      <div class=" field">
+        <h4>Email {{model.email}} </h4>
+      </div>
+    </div>
+    <div class="twelve wide column">
+      <div class="field">
+        <label class="required" for="name">{{t 'Name'}}</label>
+        {{input type='text' id='name' value=model.firstName}}
+      </div>
+      <div class="field">
+        <label class="required" for="last_name">{{t 'Family Name'}}</label>
+        {{input type='text' id='last_name' value=model.lastName}}
+      </div>
+      <div class="field">
+        <label for="description">{{t 'Details about the user'}}</label>
+        {{widgets/forms/rich-text-editor id='description' value=model.details}}
+      </div>
+      <div class="ui left aligned segment basic">
```

Comment:
Don't put the button inside a segment. That is what is giving it an unwanted left padding/margin.